numpy/core/src/umath/simd.inc.src

```diff
@@ -3561,100 +3374,15 @@ AVX512F_absolute_@TYPE@(@type@ * op,
  **                           BOOL LOOPS
  *****************************************************************************
  */
-
-/**begin repeat
- * # kind = logical_or, logical_and#
- * # and = 0, 1#
- * # op = ||, &&#
- * # sc = !=, ==#
- * # vpre = _mm*2#
- * # vsuf = si128*2#
- * # vtype = __m128i*2#
- * # type = npy_bool*2#
- * # vload = _mm_load_si128*2#
- * # vloadu = _mm_loadu_si128*2#
- * # vstore = _mm_store_si128*2#
- */
-
-/*
- * convert any bit set to boolean true so vectorized and normal operations are
- * consistent, should not be required if bool is used correctly everywhere but
- * you never know
- */
-#if !@and@
-static NPY_INLINE @vtype@ byte_to_true(@vtype@ v)
+static NPY_INLINE __m128i byte_to_true(__m128i v)
```

Comment:
other functions has transformed by using universal simd , why leave this function?

Reply:
I left it to not break the build of '_simd.inc.src', since it still been used by `sse2_absolute_BOOL`